### PR TITLE
update client.go

### PR DESCRIPTION
**Pull request by vearne**
This change is [Reviewable](https://reviewable.io/reviews/elliotchance/redismock/21)

### CODE REVIEW

**Comment by vearne** [comment]
I want a reset function.

File: client.go [hunk 1]
```diff
@@ -48,3 +48,10 @@ func (m *ClientMock) hasStub(method string) bool {
 
 	return false
 }
+
+func (m *ClientMock) Reset() {
```
Comment (elliotchance) [inline client.go:52]: I don't see the need for `Reset` in this package. It's best to create a new mock each time you need to set the expectations (for example, per test). If you need to do something more ticky than that you should handle it in your test code.

File: client_test.go [hunk 2]
```diff
@@ -5,7 +5,7 @@ import (
 	"testing"
 
 	"github.com/alicebob/miniredis"
-	"github.com/elliotchance/redismock"
+	"github.com/vearne/redismock"
```
Comment (elliotchance) [inline client_test.go:8]: Tip: This means you have the package cloned in the wrong location - you shouldn't need to change imports.

File: client.go (hunk 1 above)
Comment (vearne) [reply client.go:52]: ok, tks

**Pull request closed without merging**